_test/test_gen_sqw_for_powders/gen_sqw_powder_test_helper.m

```diff
@@ -1,4 +1,4 @@
-function gen_sqw_powder_test (spe_file, par_file, sqw_file, efix, emode)
+function gen_sqw_powder_test_helper (spe_file, par_file, sqw_file, efix, emode)
```

Comment:
we're not using the return values of this function (or those of `gen_sqw_cylinder_test_helper`)